src/NServiceBus.Core/Routing/MessageDrivenSubscriptions/Publishers.cs

```diff
@@ -103,12 +103,12 @@ public void Add(Assembly eventAssembly, string eventNamespace, string publisher)
         }
 
         /// <summary>
-        /// Adds a dynamic rule.
+        /// Registers a rule that is invoked for each subscription to determine the address of the publisher.
         /// </summary>
-        /// <param name="dynamicRule">The rule.</param>
-        public void AddDynamic(Func<Type, PublisherAddress> dynamicRule)
+        /// <param name="rule">The function that returns the publisher's physical address for a given event type, or null if there is no publisher.</param>
+        public void AddDynamic(Func<Type, PublisherAddress> rule)
```

Comment:
There we a lot of changes here that I had to deal with when I rebased, looks like I missed the actual method rename, oops!

Will fix.